metricbeat/module/system/socket/_meta/docs.asciidoc

```diff
@@ -20,8 +20,8 @@ metricbeat.modules:
 `period` value than the other metricsets.
 
 The metricset reports the process that has the socket open. In order to provide
-this information, Metricbeat must be running as root. Root access is also
-required to read the file descriptor information of other processes.
+this information for all processes in the system, Metricbeat must be run with
```

Comment:
We may want to specify that this is required if we're running in docker/not as root, lest users get confused and think there's something special they need to do if they're running a 'normal' install from an RPM or something.